### PR TITLE
Remove unnecessary line breaks

### PR DESCRIPTION
**Pull request by MZAWeb**
Opening quotes in one line and having the content in the next adds a line break in the final html which is fine in most cases but it breaks when going through some output buffering (batcache, etc)

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Whitespace found at end of line | [Linked_Posts.php:937](https://github.com/moderntribe/the-events-calendar/blob/f2f821c620b199fdda7a2395e869257ede87a25a/src/Tribe/Linked_Posts.php#L937)
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Linked_Posts.php:942](https://github.com/moderntribe/the-events-calendar/blob/f2f821c620b199fdda7a2395e869257ede87a25a/src/Tribe/Linked_Posts.php#L942)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Linked_Posts.php [hunk 1]
```diff
@@ -934,14 +934,12 @@ public function saved_linked_post_dropdown( $post_type, $current = null ) {
 				type="hidden"
 				class="tribe-dropdown linked-post-dropdown"
 				name="' . esc_attr( $name ) . '"
-				id="saved_' . esc_attr( $post_type ) . '"
-				data-placeholder="' . $placeholder . '"
-				data-search-placeholder="' . $search_placeholder . '" ' .
+				id="saved_' . esc_attr( $post_type ) . '" data-placeholder="' . $placeholder . 
+				'" data-search-placeholder="' . $search_placeholder . '" ' .
```
Comment (barryhughes) [inline src/Tribe/Linked_Posts.php]: Hey there @MZAWeb - this places the closing quote for the `data-placeholder` attribute on a different line ... is that intentional ... seems like maybe it isn't based on the PR description?

**Comment by barryhughes** [comment]
Changing base so we can include in the MR if possible.

File: src/Tribe/Linked_Posts.php (hunk 1 above)
Comment (MZAWeb) [reply src/Tribe/Linked_Posts.php]: No, that's definitely a mistake. Not sure what happened. The one below is the fix.

Sorry about that.

**Comment by barryhughes** [comment]
:ticket: [#92020](https://central.tri.be/issues/92020)

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:warning: | Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')' | [Linked_Posts.php:924](https://github.com/moderntribe/the-events-calendar/blob/2e99b0e4f0f761f9fff8ab48bc463322c16c926d/src/Tribe/Linked_Posts.php#L924)

> via jenkins/codesniffer-phpcs

**Comment by barryhughes** [comment]
Added changelog entry, updated per D's notes. Merging.

**Pull request merged**